src/rc/start-stop-daemon.c

```diff
@@ -790,6 +803,15 @@ int main(int argc, char **argv)
 			eerrorx("%s: ioprio_set %d %d: %s", applet,
 			    ionicec, ioniced, strerror(errno));
 
+		if (oom_score_adj != 0) {
+			fp = fopen("/proc/self/oom_score_adj", "w");
+			if (! fp)
```

Comment:
omit the space after the `!`

Reply:
Should I change the other occurrence(s) of this at the same time? It's not my style; I was mimicking what's already there.

Reply:
sure, if you want to adjust that by itself, sounds fine

Reply:
17e7ae7fc635fda1ecc447a6d63ef07ad2558717